sklearn/model_selection/_search.py

```diff
@@ -982,6 +984,34 @@ class GridSearchCV(BaseSearchCV):
      'split2_test_score', ...
      'std_fit_time', 'std_score_time', 'std_test_score']
 
+    >>> from sklearn import svm, datasets
```

Comment:
This rather belongs in doc/modules/grid_search.rst or 
../cross_validation.rst or doc/glossary.rst if not covered in those already